types/certstream/index.d.ts

```diff
@@ -0,0 +1,16 @@
+declare namespace CertStreamClient {
+    type Callback = (message: any, context: any) => void;
+}
+
+declare class CertStreamClient {
+    context: any;
+    callback: CertStreamClient.Callback;
+    skipHeartbeats: boolean;
+
+    constructor(callback: CertStreamClient.Callback, skipHeartbeats?: boolean);
+
+    connect(url?: string): void;
+}
+
+export = CertStreamClient;
+export as namespace CertStream;
```

Comment:
Usually the namespace is the same name as the other stuff, but maybe `CertStreamClient` is fine?

Reply:
Fixed the naming.